doc/rbd/rbd-snapshot.rst

```diff
@@ -206,107 +232,131 @@ performed these steps, you can begin cloning the snapshot.
-before  you can clone it. ::
+To clone a snapshot, specify the parent pool, the image, and the snapshot; and
+also the child pool together with the image name. You must protect the snapshot
+before you can clone it:
+
+.. prompt:: bash $
 
-	rbd clone {pool-name}/{parent-image}@{snap-name} {pool-name}/{child-image-name}
+   rbd clone {pool-name}/{parent-image}@{snap-name} {pool-name}/{child-image-name}
 	
-For example:: 
+For example:
 
-	rbd clone rbd/my-image@my-snapshot rbd/new-image
+.. prompt:: bash $
+
+   rbd clone rbd/my-image@my-snapshot rbd/new-image
 	
-.. note:: You may clone a snapshot from one pool to an image in another pool. For example, 
-   you may maintain read-only images and snapshots as templates in one pool, and writeable
-   clones in another pool.
+
+.. note:: You may clone a snapshot from one pool to an image in another pool.
+   For example, you may maintain read-only images and snapshots as templates in
+   one pool, and writeable clones in another pool.
 
 Unprotecting a Snapshot
 -----------------------
 
-Before you can delete a snapshot, you must unprotect it first. Additionally,
+Before you can delete a snapshot, you must first unprotect it. Additionally,
 you may *NOT* delete snapshots that have references from clones. You must
-flatten each clone of a snapshot, before you can delete the snapshot. :: 
+flatten each clone of a snapshot before you can delete the snapshot:
+
+.. prompt:: bash $
+
+   rbd snap unprotect {pool-name}/{image-name}@{snapshot-name}
 
-	rbd snap unprotect {pool-name}/{image-name}@{snapshot-name}
+For example:
 
-For example::
+.. prompt:: bash $
 
-	rbd snap unprotect rbd/my-image@my-snapshot
+   rbd snap unprotect rbd/my-image@my-snapshot
 
 
 Listing Children of a Snapshot
 ------------------------------
 
-To list the children of a snapshot, execute the following::
+To list the children of a snapshot, execute the following:
 
-	rbd children {pool-name}/{image-name}@{snapshot-name}
+.. prompt:: bash $
 
-For example::
+   rbd children {pool-name}/{image-name}@{snapshot-name}
 
-	rbd children rbd/my-image@my-snapshot
+For example:
+
+.. prompt:: bash $
+
+   rbd children rbd/my-image@my-snapshot
 
 
 Flattening a Cloned Image
 -------------------------
 
 Cloned images retain a reference to the parent snapshot. When you remove the
-reference from the child clone to the parent snapshot, you effectively "flatten"
-the image by copying the information from the snapshot to the clone. The time
-it takes to flatten a clone increases with the size of the snapshot. To delete 
-a snapshot, you must flatten the child images first. ::
+reference to the parent snapshot from the child clone, you effectively
+"flatten" the image by copying the information from the snapshot to the clone.
+The time it takes to flatten a clone increases with the size of the snapshot.
+To delete a snapshot, you must first flatten the child images:
```

Comment:
"you must first flatten the child images (or delete them)"

Reply:
Done.